fedireads/templates/import_status.html

```diff
@@ -0,0 +1,66 @@
+{% extends 'layout.html' %}
+{% load fr_display %}
+{% load humanize %}
+{% block content %}
+<div id="content">
+    <div>
+        <h1>Import Status</h1>
+
+        <p>
+        Import started: {{ job.created_date | naturaltime }}
+        <p>
+        {% if task.ready %}
+        Import completed: {{ task.date_done | naturaltime }}
+        {% if task.failed %}
+        <h3><span style="background-color: #ffaaaa;">TASK FAILED</span></h3>
+        <p>
+        {{ task.info }}
+        {% endif %}
+
+        {% if job.import_status %}
+            {% include 'snippets/status.html' with status=job.import_status %}
+        {% endif %}
+        {% else %}
+        Import still in progress.
+        <p>
+        (Hit reload to update!)
+        {% endif %}
+
+        <table>
+            <tr>
+                <th>
+                </th>
+                <th>
+                    Title
+                </th>
+                <th>
+                    Author
+                </th>
+                <th>
+                    Book
+                </th>
+            </tr>
+            {% for item in items %}
+            <tr>
+                <td>
```

Comment:
I think it would make more sense visually to have this column at the end and the covers column here

Reply:
Fixed.